lib/Github/Api/Enterprise/ManagementConsole.php

```diff
@@ -11,13 +11,13 @@ class ManagementConsole extends AbstractApi
      *
      * @link https://developer.github.com/v3/enterprise/management_console/#check-configuration-status
      *
-     * @param string $hash md5 hash of your license
+     * @param string $password the password for the management console
      *
      * @return array array of configuration status information
      */
-    public function configcheck($hash)
+    public function configcheck($password)
     {
-        return $this->getWithLicenseHash('/setup/api/configcheck', $hash);
+        return $this->getWithLicenseHash('/setup/api/configcheck', $password);
```

Comment:
the method name ``getWithLicenseHash`` should be changed though (as it is protected, the existing name should also be kept but deprecated and forwarding to the new method), as it is not using a license hash anymore